packages/eslint-plugin-x/lib/configs/jsdoc.js

```diff
@@ -6,26 +6,36 @@ module.exports = {
     rules: {
       'jsdoc/check-alignment': 'error',
       'jsdoc/check-indentation': 'error',
-      'jsdoc/check-param-names': 'error',
+      'jsdoc/check-param-names': 'warn',
       'jsdoc/check-tag-names': [
         'error',
         {
           definedTags: ['internal', 'remarks', 'typeParam', 'packageDocumentation']
         }
       ],
+      'jsdoc/empty-tags': 'warn',
       'jsdoc/implements-on-classes': 'off',
+      'jsdoc/multiline-blocks': 'warn',
       'jsdoc/newline-after-description': 'error',
+      'jsdoc/no-multi-asterisks': 'warn',
       'jsdoc/require-description': 'error',
-      'jsdoc/require-description-complete-sentence': 'error',
+      'jsdoc/require-description-complete-sentence': 'warn',
       'jsdoc/require-hyphen-before-param-description': 'error',
-      'jsdoc/require-param': 'error',
-      'jsdoc/require-param-description': 'error',
+      'jsdoc/require-param': 'warn',
+      'jsdoc/require-param-description': 'warn',
       'jsdoc/require-param-name': 'error',
       'jsdoc/require-param-type': 'off',
       'jsdoc/require-returns': 'error',
       'jsdoc/require-returns-check': 'error',
       'jsdoc/require-returns-description': 'error',
       'jsdoc/require-returns-type': 'off',
+      'jsdoc/tag-lines': [
+        'error',
+        'any',
+        {
+          dropLines: 'true'
```

Comment:
Isn't this option `dropEndLines` ([docs](https://github.com/gajus/eslint-plugin-jsdoc#dropendlines-defaults-to-false))?

```suggestion
          dropEndLines: 'true'
```